### PR TITLE
chore(website): [playground] inline visual editor instead of showing modal

### PR DESCRIPTION
**Pull request by armano2**
## PR Checklist

- [x] Addresses an existing open issue: fixes #000
- [x] That issue was marked as [accepting prs](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aopen+is%3Aissue+label%3A%22accepting+prs%22)
- [x] Steps in [Contributing](https://typescript-eslint.io/contributing) were taken

## Overview

- correct issues with re-rendering and constant updating of visual editor
- move visual editor to be displayed in place of text editor

ref https://github.com/typescript-eslint/typescript-eslint/pull/6656#discussion_r1141374222

### CODE REVIEW

File: packages/website/src/components/Playground.tsx [hunk 1]
```diff
@@ -70,6 +55,25 @@ function Playground(): JSX.Element {
...
+  const activeVisualEditor = useMemo(() => {
+    if (!isLoading) {
+      return visualEslintRc && activeTab === 'eslintrc'
+        ? 'eslintrc'
+        : visualTSConfig && activeTab === 'tsconfig'
+        ? 'tsconfig'
+        : undefined;
+    }
+    return undefined;
+  }, [activeTab, isLoading, visualEslintRc, visualTSConfig]);
```
Comment (JoshuaKGoldberg) [inline packages/website/src/components/Playground.tsx:67]: [Performance] I don't think this is perf-intensive enough to justify a `useMemo`?

**Review by JoshuaKGoldberg — APPROVED** [review]
*YES* so hyped for this!! Love it!

File: packages/website/src/components/Playground.tsx (hunk 1 above)
Comment (armano2) [reply packages/website/src/components/Playground.tsx:67]: agree, initially there was way more code in there :)

**Comment by armano2** [comment]
hmm, i noticed some race condition when selecting items, also hover seem to be not working on even items

https://user-images.githubusercontent.com/625469/229460061-3e4b0082-01f4-4f75-92cb-7f835f98ef30.mp4

**Pull request merged**